code/modules/client/preference/loadout/loadout_general.dm

```diff
@@ -18,22 +18,91 @@
 	display_name = "a pack of Midoris"
 	path = /obj/item/storage/fancy/cigarettes/cigpack_midori
 
+/datum/gear/marijuanapack
```

Comment:
There's no exception in space law for medical use, which means this lets crew spawn with an item that's technically illegal?

Reply:
didn't realize this had been commented on. One could say the player smuggled it aboard but if need be I can remove it

Reply:
I don't see why spawning with a slightly illegal item is an issue, personally. not like sec even enforces that law unless the round is very slow.

Reply:
This is a free healing item so, I'm going to have to say "probably not" on this one.

Unless there's a subtype that has the cannabidiol removed from them.